packages/apps/job-launcher/server/src/common/interceptors/snake-case.ts

```diff
@@ -21,8 +22,13 @@ export class SnakeCaseInterceptor implements NestInterceptor {
       request.query = CaseConverter.transformToCamelCase(request.query);
     }
 
-    return next
-      .handle()
-      .pipe(map((data) => CaseConverter.transformToSnakeCase(data)));
+    return next.handle().pipe(
+      map((data) => {
+        if (data instanceof StreamableFile) {
```

Comment:
W/o this files can't be downloaded: `StreamableFile` gets modified to snake case and just returned as `json`